### ISSUE TITLE
Error in Mission Planning from QGC

### ISSUE THREAD

**Issue by SaliniVenate** [intel-aero/meta-intel-aero#213]
While uploading a mission with RTL option to Intel Aero from QGC I get the following error:
1) Error during Mission communication with Vehicle: Vehicle returned error: Command not supported (MAV_MISSION_UNSUPPORTED).Command(,,20)

![inel_aero_error](https://user-images.githubusercontent.com/25497118/28159461-a25241ee-67da-11e7-9c7e-d30708316328.png)

2) When I  un-check the RTL checkbox, the mission is sent successfully to vehicle but when "Start Mission" command is given it shows error:Unable to start mission:Vehicle not ready.

![vehicle_not_ready](https://user-images.githubusercontent.com/25497118/28159372-598cc2a4-67da-11e7-943d-450585bf34f2.png)

These operations work fine  while testing on jMAVSim SITL which is running PX4 version 1.6.5dev and QGC compiled from master branch.

I'm using Intel aero board with intel-aero image 1.4 flashed on it.
BIOS_VERSION = Aero-01.00.12_Prod
OS_VERSION = Poky Aero (Intel Aero Linux Distro) 1.5.0-dev (pyro)"
AIRMAP_VERSION = 1.8
FPGA_VERSION = 0xc0

The firmware version as showed by QGC is 1.6.0dev.
![firmware_version](https://user-images.githubusercontent.com/25497118/28159559-f7882d36-67da-11e7-9370-04fdafe91ad5.png)

**Comment by sugnanprabhu** [comment]
@SaliniVenate 
The latest aero release is using v1.6.3 firmware.
Can you please try updating the px4 firmware on the aero with the following command and let us know if you still face the error.
# aerofc_update.sh /etc/px4-fw/nuttx-aerofc-v1-default.px4

**Comment by anselmolsm** [comment]
> OS_VERSION = Poky Aero (Intel Aero Linux Distro) 1.5.0-dev (pyro)"
It is not 1.4.

> The firmware version as showed by QGC is 1.6.0dev.
Please follow what @sugnanprabhu suggested. That will flash 1.6.3 into Aero FC. Then repeat the test and post results here. Thanks.

**Comment by lucasdemarchi** [comment]
There are lots of changes from v1.6.3 to v1.6.5. We are likely going to update once it's flight tested.  Can you test this firmware on aero? http://px4-travis.s3.amazonaws.com/Firmware/stable/aerofc-v1_default.px4

**Comment by SaliniVenate** [comment]
I tried the updating px4 the way @sugnanprabhu  suggested and the issue 1 is solved. I'm able to send the mission successfully to vehicle. @lucasdemarchi , I tested the firmware you mentioned on intel aero and with that also I did not find any issue with sending a mission to the vehicle. In both cases the firmware version as displayed in QGC is 1.6.2dev.

The second issue  I believe is because the vehicle is not able to connect GPS as I'm working inside the lab.
